projects/com.oracle.truffle.llvm.nodes.impl/src/com/oracle/truffle/llvm/nodes/impl/base/integers/LLVMI32Node.java

```diff
@@ -36,6 +36,7 @@
 
     public static final long MASK = 0xffffffffL;
     public static final int BYTE_SIZE = 4;
+    public static final int BIT_SIZE = 32;
```

Comment:
As you can see, BYTE_SIZE was already there, and it would need many changes to rename it to SIZE_IN_BYTES. Maybe it's better to leave it as it is for now, and rename it in a later PR?